go.mod

```diff
@@ -9,8 +9,8 @@ retract (
 
 require (
 	github.com/google/go-cmp v0.5.9
-	golang.org/x/net v0.17.0
+	golang.org/x/net v0.21.0
```

Comment:
Related to release: https://github.com/golang/net/commit/c63010009c802314a29324ce49987897f9838e29